docs/adr/0021-move-verification-tests.md

```diff
@@ -0,0 +1,28 @@
+# 21. Move verification testing to its own GitHub workflow
+
+Date: 2024-08-12
+
+## Status
+
+Accepted
+
+## Context
+
+Regular CI runs (on push, PR etc.) ran all tests, including the verification tests.
+
+These verification tests make requests to the GitHub API (and the OpenCodelists API).
```

Comment:
It might be worth explaining what the verification tests are used for, so something like:

> We use verification tests to ensure that our understanding of external APIs is still correct and that our fake API objects, used in our unit tests, are still valid. These verification tests make requests to the GitHub API (and the OpenCodelists API).

Reply:
👍